### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AlperMumcular**
## Summary by CodeRabbit

- **Documentation**
	- Updated the README file with additional text.

### CODE REVIEW

**Comment by vIremAydin** [comment]
<p>dvdfvsfv</p>

**Comment by vIremAydin** [comment]
<h1>kocaman review</h1>

**Comment by aysekelleci** [comment]
<p>deneme</p>

**Comment by aysekelleci** [comment]
<p>test</p>

**Comment by aysekelleci** [comment]
<p>deneme</p>

**Comment by aysekelleci** [comment]
**ACTIVE**: <p>deneme</p>

**Comment by vIremAydin** [comment]
> <<p>fdgupdate comment 22</p>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/11#issuecomment-2017785561 

manm

**Comment by vIremAydin** [comment]
> <p>dvdfvsfv</p>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/11#issuecomment-2016826047 

https://github.com/hubreviewapp/hubreviewapp.github.io/pull/11#issuecomment-2016826047

**Comment by vIremAydin** [comment]
> <<p>fdgupdate comment 22</p>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/11#issuecomment-2017785561 

kkk

**Comment by vIremAydin** [comment]
> 
> <<p>fdgupdate comment 22</p>
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/11#issuecomment-2017785561 

kkk
> https://github.com/hubreviewapp/hubreviewapp.github.io/pull/11#issuecomment-2042952633

File: README.md [hunk 1]
```diff
@@ -11,3 +11,4 @@ On systems lacking Adobe Acrobat, `qpdf` may be used to encrypt PDFs:
 for f in *.pdf; do qpdf --encrypt [pass] [pass] 256 -- "$f" "protected-$f"; done
```
Comment (Ece-Kahraman) [inline README.md:11]: suggestion(non-blocking): sup

**Review by Ece-Kahraman — COMMENTED** [review]
hello

**Comment by aysekelleci** [comment]
**ACTIVE**: <p>csdn</p>

**Comment by aysekelleci** [comment]
**ACTIVE**: <p>denee</p>

**Pull request closed without merging**